x-pack/plugins/observability_solution/slo/kibana.jsonc

```diff
@@ -49,7 +49,8 @@
       "kibanaUtils",
       "unifiedSearch",
       "embeddable",
-      "ingestPipelines"
+      "ingestPipelines",
+      "dashboard"
```

Comment:
@jloleysens Thanks for the review! I have a question. As part of this [PR], (https://github.com/elastic/kibana/pull/182120), a `registerDashboardPanelPlacementSetting` was added to the `dashboard` plugin, which I import in the slo plugin like this.

```
import {
  registerDashboardPanelPlacementSetting
} from '@kbn/dashboard-plugin/public';
```

All I had to do, was to add `dashboard` to the required bundles of the slo plugin. I didn't add it to the `requiredPlugins` and all works fine. Maybe I should add it to the requiredPlugins as well?

Reply:
We discussed about it offline and for now it is ok `only in the requiredBundles`